src/Logary.Prometheus.Tests/Program.fs

```diff
@@ -0,0 +1,122 @@
+﻿// Learn more about F# at http://fsharp.org
+
+open System
+open System.Net
+open System.Threading
+open Expecto
+open Logary
+open Logary.Prometheus
+open Logary.Metric
+open Expecto.Flip
+open System.Net.Http
+open Hopac
+
+module Exporter =
+
+  open Suave
+
+
+  let checkTextProtocol expected actual =
+    expected |> String.split '\n' |> Expect.containsAll "should exported to prometheus text protocol" (actual |> String.trimStart |> String.split '\n')
+
+
+  [<Tests>]
+  let tests = testList "Logary.Prometheus" [
+
+
+    testCaseJob "export to prometheus text protocol" <| job {
+      let httpClient = new HttpClient()
+      httpClient.BaseAddress <- Uri("http://127.0.0.1:8080/")
+      let exportedTextJob = Job.liftTask (fun (endpoint:string) -> httpClient.GetStringAsync(endpoint)) "metrics"
+
+      let webConfig = defaultConfig.withBindings [HttpBinding.create HTTP IPAddress.Loopback 8081us]
+
+      let cts = new CancellationTokenSource()
+      use cancelWebServer =  { new IDisposable with member x.Dispose() = cts.Cancel () }
+
+      ExporterConf.create "/metrics" LogManager.DefaultMetricRegistry
+      |> Exporter.runAsync cts.Token
+      |> ignore
+
+      let gauge =
+        GaugeConf.create {name = "time_latancy"; description = "test time latancy"; labelNames = [| "endpoint" |]}
+        |> GaugeConf.withHistogram [|50.; 100.; 500.;|]
+        |> LogManager.DefaultMetricRegistry.registerMetric
```

Comment:
Do you think we could write this test on a logary instance? I'd prefer to have test non-interfering with each other. And this is a global, right?

Reply:
Yes, you are right. fixed.